lib/callback_model.js

```diff
@@ -39,8 +44,13 @@ class CallbackModel extends EventEmitter {
     return ch;
   }
 
-  createConfirmChannel (cb) {
+  createConfirmChannel (options, cb) {
+    if (arguments.length === 1) {
+      cb = options;
+      options = undefined;
+    }
     var ch = new ConfirmChannel(this.connection);
+    ch.setOptions(options);
```

Comment:
I used a setter to specify the channel options to avoid making a breaking change to the Channel constructor